synapse/res/templates/sso_auth_account_details.html

```diff
@@ -138,7 +138,7 @@ <h1>Create your account</h1>
         <div class="username_input" id="username_input">
           <label for="field-username">Username (required)</label>
           <div class="prefix">@</div>
-          <input type="text" name="username" id="field-username" value="{{ user_attributes.localpart }}" autofocus>
+          <input type="text" name="username" id="field-username" value="{{ user_attributes.localpart }}" autofocus autocorrect="off" autocapitalize="none">
```

Comment:
I see `autocorrect` at https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#non-standard_attributes, but not `autocapitalize`. Do you have a reference for that?

Reply:
Yep, theres one over at Apple here: https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/SafariHTMLRef/Articles/Attributes.html#//apple_ref/doc/uid/TP40008058-autocapitalize